indexer-app/src/cmr/indexer/data/elasticsearch.clj

```diff
@@ -209,30 +209,13 @@
 
 (defn- non-tombstone-concept->bulk-elastic-doc
   "Takes a non-tombstoned concept map (a normal revision) and returns an elastic document suitable
-   with ttl fields for bulk indexing."
+   for bulk indexing."
   [context concept]
   (let [parsed-concept (cp/parse-concept context concept)
         delete-time (get-in parsed-concept
                             [:data-provider-timestamps :delete-time])
-        now (tk/now)
-        ttl (when delete-time
-              (if (t/after? delete-time now)
-                (t/in-millis (t/interval now delete-time))
-                0))
-        elastic-doc (parsed-concept->elastic-doc context concept parsed-concept)
-        elastic-doc (if ttl
-                      (assoc elastic-doc :_ttl ttl)
-                      elastic-doc)]
-    (if (or (nil? ttl)
-            (> ttl 0))
-      elastic-doc
-      (info
-       (str
-        "Skipping expired concept ["
-        (:concept-id concept)
-        "] with delete-time ["
-        (f/unparse (f/formatters :date-time) delete-time)
-        "]")))))
+        elastic-doc (parsed-concept->elastic-doc context concept parsed-concept)]
+    elastic-doc))
```

Comment:
nit: no need to define `elastic-doc` in the let block, just return the value here.